pkgs/development/python-modules/restinstance/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, fetchFromGitHub, buildPythonPackage, jsonschema, nox }:
+
+buildPythonPackage rec {
+  pname = "RESTinstance";
+  version = "1.3.0";
```

Please add `format` and I also suggest to limit new modules to supported Python releases.

```suggestion
  version = "1.3.0";
  format = "setuptools";

  disabled = pythonOlder "3.7";
```